ide/spi.debugger.ui/src/org/netbeans/modules/debugger/ui/DebuggerManagerListener.java

```diff
@@ -673,7 +673,11 @@ private static void doCloseDebuggerUI() {
             final Set<Component> initiallyOpenedMinimized;
             if (componentsInitiallyOpened.isEmpty()) {
                 initiallyOpened = Collections.emptyList();
+<<<<<<< HEAD
                 initiallyOpenedMinimized = Collections.EMPTY_SET;
+=======
+                initiallyOpenedMinimized = Collections.emptySet();
+>>>>>>> [NETBEANS-3377] - cleanup unchecked conversions
```

Comment:
This is a merge conflict, that was not resolved - please check again.

Reply:
UGG!! Must have missed something. Sorry about that.. Should be fixed down..